server/rest/src/test/java/org/infinispan/rest/search/NonSharedIndexSearchTest.java

```diff
@@ -22,8 +22,8 @@ ConfigurationBuilder getConfigBuilder() {
       return builder;
    }
 
-   @Override
```

Comment:
It seems that removing ```@Override``` was not really needed.

Reply:
fixed